molgenis-navigator/src/main/java/org/molgenis/navigator/download/job/DownloadJobExecution.java

```diff
@@ -0,0 +1,32 @@
+package org.molgenis.navigator.download.job;
+
+import javax.annotation.Nullable;
+import org.molgenis.data.Entity;
+import org.molgenis.data.meta.model.EntityType;
+import org.molgenis.jobs.model.JobExecution;
+
+public class DownloadJobExecution extends JobExecution {
+  public DownloadJobExecution(Entity entity) {
+    super(entity);
+    setType(DownloadJobExecutionMetaData.DOWNLOAD_JOB_TYPE);
+  }
+
+  public DownloadJobExecution(EntityType entityType) {
+    super(entityType);
+    setType(DownloadJobExecutionMetaData.DOWNLOAD_JOB_TYPE);
+  }
+
+  public DownloadJobExecution(String identifier, EntityType entityType) {
+    super(identifier, entityType);
+    setType(DownloadJobExecutionMetaData.DOWNLOAD_JOB_TYPE);
+  }
+
+  public void setResources(String resourceJson) {
```

Comment:
How about:
```
  public void setResources(List<Resource> resources>) {
   ...
  }

  public List<Resource> getResources() {
   ...
  }
```

This would simplify creating/using/testing the NavigatorServiceImpl. The JSON <--> resource translation could be moved to a util class that can be used by both the copy and download job (@tommydeboer).